src/extension/ln/nostr/index.ts

```diff
@@ -3,11 +3,19 @@ import { Event } from "./types";
 export default class NostrProvider {
   nip04 = new Nip04(this);
 
+  private async enable() {
+    return await this.execute<{
+      enabled: boolean;
+      remember: boolean;
+    }>("enable");
+  }
+
   async getPublicKey(): Promise<string> {
     return await this.execute("getPublicKeyOrPrompt");
   }
 
-  async signEvent(event: Event): Promise<Event> {
+  async signEvent(event: Event): Promise<Event | boolean> {
+    await this.enable();
```

Comment:
guess you'd need this now in all the other calls also.

Reply:
Would do `getPublicKey` in a separate PR